templates/vue/src/components/Lightbox/media/MultiContentMedia/index.vue

```diff
@@ -108,6 +149,20 @@ export default {
       "isMultiContent",
     ]),
     ...mapState(["favourites", "rootId"]),
+    parentNode() {
+      const parentNodeId = this.getParent(this.node.id)
+      return this.getNode(parentNodeId)
+    },
+    filteredPages() {
+      return this.pages
+        ? this.pages.filter(page => page.accessible || !page.hideWhenLocked)
```

Comment:
Since all the other checks of this form look like `node.unlocked || !node.hideWhenLocked`, would it be better to use `page.unlocked` here for consistency?